lib/checks/color/color-contrast.js

```diff
@@ -1,6 +1,12 @@
+var useScroll = true;
+
+if((options || {}).noScroll) {
```

Comment:
@dylanb you had suggested using negation here. Were you thinking initializing `useScroll` to false on line 1, then setting it to `true` if this expression is truthy?